css/typography.styl

```diff
@@ -41,7 +41,7 @@ h6
   font-weight: bold
   margin: 0
 
-.on-home-page, .on-landing-page, .on-home-page-not-logged-in
+.on-home-page, .on-landing-page, .on-home-page-not-logged-in, .on-home-page-logged-in
```

Comment:
Same comment here, can we just use `on-home-page` and make sure the root has that class applied?